app/services/gemini.rb

```diff
@@ -1,5 +1,5 @@
 module Gemini
-  @scope = "account:read,balances:read,payments:send,payments:read"
+  @scope = "balances:read,account:read,payments:create"
```

Comment:
why is `payments:create` needed? i would think we need read permission to assemble a statement for the user in the future

Reply:
We need payments:create in order to generate new recipientIds, as well as make changes to default currency